core/scripts/Voting.js

```diff
@@ -157,17 +176,17 @@ async function fetchCryptoComparePrice(request) {
   }
 
   const tradeTime = jsonOutput.Data[0].time;
-  console.log(`Retrieved quote [${price}] at [${tradeTime}] for asset [${identifier.first}${identifier.second}]`);
+  // console.log(`Retrieved quote [${price}] at [${tradeTime}] for asset [${identifier.first}${identifier.second}]`);
```

Comment:
I thought more about the suppressed logs -- these logs were added to help users who are running the AVS. I think my biggest concern is that the UX is really bad in production if the AVS doesn't log anything. As for tests, I'm totally in favor of having a way to suppress logs as long as they only get suppressed in the *test environment*.

Reply:
I agree it would be convenient to suppress tests in the test environment, I'll add a flip